pre-reg-113-115/pom.xml

```diff
@@ -51,7 +51,7 @@
 		<dependency>
 			<groupId>io.mosip.kernel</groupId>
 			<artifactId>kernel-core</artifactId>
-			<version>1.2.0-SNAPSHOT</version>
+			<version>1.2.0.1-SNAPSHOT</version>
```

Comment:
Have we tested the pre-reg utitlity using this version locally or in any other enviroment ?
If we don't want to test this, will this not be safer to depend on the 1.2.0 version rather ?

Reply:
ok sure, will Test in local and update